### ISSUE TITLE
Alternatives to missing symbols?

### ISSUE THREAD

**Issue by nathgit** [molarmanful/li#1]
The following symbols are unavailable on my device (they show up as a rectangle).
⦜⦾▯⧈⦹⦀⬬⬕⦚
Are there acceptable alternatives that might show up on more phones? My phone is running android 6.0.1.

**Comment by molarmanful** [comment]
Apologies for the late reply!

I've run into this issue regularly with other Unicode-based projects. Something that I hope should work is using a font that supports a lot of Unicode characters (including the characters you've put above). [Deja Vu Sans Mono](https://dejavu-fonts.github.io) will likely work; not entirely sure of an interface/site/etc. that would make it easier to view and interact with the glyphs, but I suppose it's a place to start.

It's been an extremely long time since I've last worked on this project, but I might revisit it because it's got some potential usefulness in another project I'm conceptualizing. Perhaps I'll create a version of the symbols using a reduced codepage, or something along those lines.

**Comment by nathgit** [comment]
There's no way to add/change system fonts without root though as far as I know. If it's possible with ADB that's still not an option because I use a chromebook.

**Comment by molarmanful** [comment]
Yeah, I figured, the solution I was thinking of would be web-based -- kind of like an IDE but tailor-made for `"`. The fonts can be loaded in pretty trivially.
